packages/database/src/modelClasses/DashboardItem.js

```diff
@@ -25,6 +25,14 @@ export class DashboardItemModel extends DatabaseModel {
           comparator: '&&', // User has ANY of the permission groups
           comparisonValue: permissionGroups,
         },
+        project_codes: {
+          comparator: '@>',
+          comparisonValue: projectCodes,
+        },
+        entity_types: {
+          comparator: '@>',
+          comparisonValue: entityTypes,
+        },
```

Comment:
I think everything we're filtering on is actually attached to the `dashboard_relation` table, so probably good to make these explicitly `dashboard_relation.project_codes` etc